codegen/src/main/twirl/templates/ScalaServer/Handler.scala.txt

```diff
@@ -128,12 +128,17 @@ object @{serviceName}Handler {
           .recoverWith(GrpcExceptionHandler.from(eHandler(system.classicSystem))(system, writer))
       ).getOrElse(unsupportedMediaType)
 
-      Function.unlift((req: model.HttpRequest) => req.uri.path match {
+
+
+      val httpHandler = akka.grpc.HttpApi.serve(@{service.name}.descriptor, handle _)
```

Comment:
This effectively elevates `HttpApi.serve` to stable API: if a project uses code generated by this version, and updates the Akka gRPC runtime library, we'll have to make sure these new versions of the runtime library still provide that `serve` function with the same signature and behavior. I'm not sure I'm comfortable with that yet.

I wonder if it would be possible to use this `HttpApi` from the main client code, so the user can optionally enable it, and we can mark it `ApiMayChange` to leave room for evolving the API further later?